### DISCUSSION TITLE
边缘滚动会受任务栏影响而失效

### DISCUSSION THREAD

**Discussion by Ludicio** [shuax/MouseInc.Settings#200] [General]
环境win10 21H1 19043.1348 企业版，mouseinc 2.11.7.B。

**开启边缘滚动，_任务栏在屏幕左侧_，禁用忽略全屏**

> 启动软件后，左侧滚动鼠标调节音量正常√

> 锁屏解锁或运行全屏程序后，左侧滚动鼠标调节音量失效×，按alt滚动鼠标调节音量失效×

**开启边缘滚动，_任务栏在屏幕下方_ ，禁用忽略全屏**

> 启动软件后，左侧滚动鼠标调节音量正常√

> 锁屏解锁或运行全屏程序后，左侧滚动鼠标调节音量正常√，按alt滚动鼠标调节音量正常√

怀疑是任务栏位置对功能唤醒产生了影响，盼修复。

**Comment by shuax** [comment]
我现在win11，设置不了在左边，但是听起来好像是有这个问题，我看看代码。

**Comment by shuax** [comment]
你是不是多显示器啊？我强行改注册表让任务栏在左边没有复现。
或者是你运行的全屏程序是管理员程序，这个时候mouseinc不能工作，需要用管理员权限运行。

**Reply by Ludicio** [reply]
是的，我用的是多显示器，侧边任务栏栏在最左面的显示器上。
全屏程序运行的时候音量调节是正常的，但是一退出音量调节就失效了。
